media-sound/easytag/files/easytag-2.4.3-fix-build-taglib2.patch

```diff
@@ -0,0 +1,202 @@
+from https://gitlab.gnome.org/GNOME/easytag/-/merge_requests/16
```

Comment:
For future reference, for PRs/commits you're grabbing, it's best to append .patch to the URL and use that, so you have the headers on top (then put a link on top of that).

(e.g. look at https://gitlab.gnome.org/GNOME/easytag/-/merge_requests/16.patch)

Reply:
done